src/routes/(console)/project-[project]/overview/platforms/components/OnboardingPlatformCard.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import type { ComponentType } from 'svelte';
+
+    export let icon: ComponentType;
+    export let iconSize: number | undefined = undefined;
+    export let iconColor: string | undefined = undefined;
+</script>
+
+<div class="outer-card">
+    <div
+        class="inner-card"
+        style="
+            --icon-color: {iconColor || 'inherit'};
+            --icon-size: {iconSize ? `${iconSize}rem` : 'inherit'};
+        ">
```

Comment:
```suggestion
        style:--icon-color={iconColor || 'inherit'}
        style:--icon-size={iconSize ? `${iconSize}rem` : 'inherit'}>
```

I think this should be possible 🤔

Reply:
cc @ItzNotABug